### ISSUE TITLE
(Optional) Test Connection LB with PgBouncer

### ISSUE THREAD

**Issue by randomorder** [geosolutions-it/eumetsat-EOWS#451]

**Comment by randomorder** [comment]
This is optional. Let's discuss it and see if we have enought time at the end of the PG HA cluster implementation

**Comment by lpasquali** [comment]
@randomorder this will only work for SSL the bouncer won't accept non-ssl connection if I do not craft the zalando's docker image, which I'd avoid as I suspect that jdbc could work fine with ssl

**Comment by randomorder** [comment]
that's fine we can configre connection pools in Tomcat to use SSL

**Comment by lpasquali** [comment]
> that's fine we can configre connection pools in Tomcat to use SSL

I missed this comment sorry, so ssl, with or without pg bouncer could be a viable possibility @randomorder?

**Comment by randomorder** [comment]
GeoServer and Tomcat are both SSL capable
I guess PgBouncer is using a self-signed cert so we'll have to play with it, eventually disabling the validation on the JNDI connection pool and (if possible) in the GS datadir in the property files or alternatively install the cert in the JVMs

**Comment by randomorder** [comment]
moving to backlog

**Comment by randomorder** [comment]
we're not delivering this now but please summarize your findings. I take this is basically working..

**Comment by lpasquali** [comment]
> we're not delivering this now but please summarize your findings. I take this is basically working..

yes it works well, but is only ssl, which is possibly not an issue as we spoke @randomorder

**Comment by simboss** [comment]
can we close thi sone?

**Comment by randomorder** [comment]
Yes
please keep though in mind that not implemented or delivered this because it was not part of the requirements
